docs/discover/search.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[search]]
 == Searching Your Data
 You can search the indices that match the current index pattern by entering
-your search criteria in the Query bar. You can perform a simple text search,
-use the Lucene https://lucene.apache.org/core/2_9_4/queryparsersyntax.html[
-query syntax], or use the full JSON-based {ref}/query-dsl.html[Elasticsearch
-Query DSL].
+your search criteria in the Query bar. You can use the Lucene
+https://lucene.apache.org/core/2_9_4/queryparsersyntax.html[
+query syntax], the full JSON-based {ref}/query-dsl.html[Elasticsearch
+Query DSL], or Kuery, an experimental new query language built specifically for Kibana.
```

Comment:
"DSL], or Kuery," I think we might have one too many comma's ==> "DSL] or Kuery,"